release/release_logs/1.13.0/benchmarks/many_actors.json

```diff
@@ -0,0 +1,15 @@
+{
+    "_peak_memory": 5.32,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n186\t1.61GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n2169\t0.88GiB\tpython distributed/test_many_actors.py\n211\t0.26GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n294\t0.12GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n76\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n404\t0.11GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session auth_start\n2214\t0.06GiB\tray::MemoryMonitorActor.run()\n77\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=63d80251\n204\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.80.251:9031 --host=0.0.0.0\n244\t0.05GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/log_m",
+    "actors_per_second": 321.72392004133843,
```

Comment:
Hmm, this has about 20% regression. However, when I checked the perf graph, this varies from 320 ~ 380. @krfricke can you double check if the previous run was faster than this?